plugins/woocommerce/tests/README.md

```diff
@@ -53,19 +58,14 @@ Example:
 
 ## Running Tests
 
-Change to the plugin root directory and type:
-
-    $ vendor/bin/phpunit
-
-The tests will execute and you'll be presented with a summary.
-
-You can run specific tests by providing the path and filename to the test class:
+Depending your environment, you will use one of these commands to run unit tests:
 
-    $ vendor/bin/phpunit tests/legacy/unit-tests/importer/product.php
+- **Recommended Environment**: `pnpm --filter=woocommerce test:unit:env`
+- **Manual Environment**: `pnpm --filter=woocommerce test:unit`
 
-A text code coverage summary can be displayed using the `--coverage-text` option:
+The tests will execute and you will be presented with a summary.
 
-    $ vendor/bin/phpunit --coverage-text
+You can run specific test files by providing the path to the test class, such as `pnpm --filter=woocommerce test:unit:env tests/legacy/unit-tests/importer/product.php`. A text code coverage summary can be displayed using the `--coverage-text` option.
```

Comment:
Is this the same as the testing instructions?

>pnpm --filter=woocommerce test:unit:env --filter=test_handle

There is no `--filter` bit.

Running `pnpm --filter=woocommerce test:unit:env tests/legacy/unit-tests/importer/product.php` produces an error for me:

```
Cannot open file "tests/legacy/unit-tests/importer/product.php".
```

Reply:
For the `--filter` it should run 10 tests instead of the thousands. You're passing `--filter=test_handle` to PHPUnit which then, well, filters.

As for the file path, you're right, that doesn't seem to work. I think it's because the working directory might not be great, let me look into that.

Reply:
It looks like the path to a specific file isn't going to work right now. We're going to need an upstream PR to set the working directory when using `wp-env run` 😓 If we brand this as experimental I think we can still keep it. It isn't used anywhere except manually.